jumpscale/packages/marketplace/frontend/components/Home.vue

```diff
@@ -84,9 +86,17 @@ module.exports = {
           return obj;
         }, {});
     },
+    getCurrentUser() {
```

Comment:
why don't we redefine the user here? cannot we pass it from parent component or get it from a shared state instead?